system-addon/content-src/styles/_activity-stream.scss

```diff
@@ -99,13 +96,14 @@ a {
     }
 
     &.dismiss {
+      background-color: transparent;
       border: 0;
       padding: 0;
       text-decoration: underline;
     }
 
     &.done {
-      background: $input-primary;
+      background-color: var(--newtab-button-primary-color);
       border: solid 1px $blue-60;
```

Comment:
not sure why I didn't use $input-primary instead of this $blue-60 for border.. but it should probably just be var(--newtab-button-primary-color) now